openai-assistant/target-platform/ivy.target

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target name="ivy">
+	<locations>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://p2.axonivy.com/p2/11.3/"/>
+			<unit id="ch.ivyteam.ivy.designer.product" version="11.3.1"/>
+			<unit id="ch.ivyteam.ivy.designer.rule.engine.feature.feature.group" version="11.3.1.202406202109"/>
+			<unit id="ch.ivyteam.ivy.designer.webeditor.feature.feature.group" version="11.3.1.202406202117"/>
+			<unit id="ch.ivyteam.ivy.elasticsearch.feature.feature.group" version="11.3.1.202406210642"/>
```

Comment:
why do we need these? I'd just adapt from the slim platform defined in the excel-importer https://github.com/axonivy-market/excel-importer/blob/master/target-platform/ivy.target

... furthermore this could be a good time to also introduce/adapt the dev-docs on how to use a target-platform, just like they exist in the excel-importer. or at least a reference to these docs.

Reply:
I updated it to align with the target for the excel-importer. I'm preparing to leave next week, and I'm writing some documentation for my team members. I'm also referencing the dev-setup guide from the excel-importer. Thank you for reviewing